tests/test_pyscalcli.py

```diff
@@ -142,17 +142,17 @@ def test_pyscal_client_static(tmp_path, caplog, default_loglevel, mocker):
             "eclipse/include/props/relperm-fam2.inc",
         ],
     )
-    pyscalcli.main()
-    assert Path("eclipse/include/props/relperm-fam2.inc").is_file()
-    assert not any(record.levelno == logging.ERROR for record in caplog.records)
+    with pytest.raises(SystemExit, match="Output directory not found"):
```

Comment:
It's possible to be a bit more pedantic and have pytest look for `IOError` rather than `SystemExit` but it doesn't make much of a different with the match there :-)

Reply:
You are right. There seems to be a few more of this in the code. So, I think I will update them all in another PR.

Thanks 😄